ambry-commons/src/main/java/com/github/ambry/commons/NettySslHttp2Factory.java

```diff
@@ -101,8 +105,8 @@ static SslContext getServerSslContext(SSLConfig config) throws GeneralSecurityEx
     return sslContextBuilder.sslProvider(SslContext.defaultClientProvider())
         .clientAuth(NettySslFactory.getClientAuth(config))
         /* NOTE: the cipher filter may not include all ciphers required by the HTTP/2 specification.
-         * Please refer to the HTTP/2 specification for cipher requirements. */
-        .ciphers(Http2SecurityUtil.CIPHERS, SupportedCipherSuiteFilter.INSTANCE)
+         * Please refer to the HTTP/2 specification for cipher requirements. */.ciphers(Http2SecurityUtil.CIPHERS,
```

Comment:
nit: is this auto format by IDEA? From my side, the original one is the auto-format result.

Reply:
reformatted